packages/rn-tester/js/examples/Accessibility/AccessibilityAndroidExample.android.js

```diff
@@ -61,22 +75,15 @@ class AccessibilityAndroidExample extends React.Component {
               <Text>Click me</Text>
             </View>
           </TouchableWithoutFeedback>
-          <Text accessibilityLiveRegion="polite">
+          <Text accessibilityLiveRegion={'polite'}>
             Clicked {this.state.count} times
           </Text>
         </RNTesterBlock>
 
         <RNTesterBlock title="Overlapping views and importantForAccessibility property">
           <View style={styles.container}>
             <TouchableWithoutFeedback
-              style={{
-                position: 'absolute',
-                left: 10,
-                top: 10,
-                right: 10,
-                height: 100,
-                backgroundColor: 'green',
-              }}
+              style={[styles.touchableContainer, this.props.style]}
```

Comment:
I noticed that `this.props.style` isn't used before your change. Is it needed? Could we remove it? Then we wouldn't have to make any changes to `TouchableWithoutFeedback`.

Reply:
sure, simplified this to below code, which i guess should work fine : 
`style={styles.touchableContainer}`